codalab/lib/path_util.py

```diff
@@ -235,12 +235,11 @@ def hash_file_contents(path):
 ################################################################################
 
 
-def copy(source_path, dest_path, follow_symlinks=False, exclude_patterns=None):
+def copy(source_path, dest_path, follow_symlinks=False):
```

Comment:
With this PR, `exclude_patterns` is no longer used in `copy`, so I removed it to simplify `copy`'s implementation.